<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Nextturn</class>
 <widget class="QDialog" name="Nextturn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Play again?</string>
  </property>
  <widget class="QPushButton" name="pbtn_ok">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_cancel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_save">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>320</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Press 'Save' to save this game.&lt;/p&gt;&lt;p&gt;Press 'OK' to start the next round of game.&lt;/p&gt;&lt;p&gt;Press 'Cancel' to quit and choose game modes.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbtn_cancel</sender>
   <signal>clicked(bool)</signal>
   <receiver>Nextturn</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbtn_ok</sender>
   <signal>clicked(bool)</signal>
   <receiver>Nextturn</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
